Submitted By: Fernando de Oliveira <famobr at yahoo dot com dot br>
Date: 2013-11-29
Initial Package Version: 25.0.1
Upstream Status: Submitted
Origin: mozilla
URL: https://bug944454.bugzilla.mozilla.org/attachment.cgi?id=8340117
Description: Fixes build with FreeType-2.5.1

diff --git a/config/system-headers b/config/system-headers
--- a/config/system-headers
+++ b/config/system-headers
@@ -408,16 +408,29 @@ freetype/ftoutln.h
 freetype/ttnameid.h
 freetype/tttables.h
 freetype/t1tables.h
 freetype/ftlcdfil.h
 freetype/ftsizes.h
 freetype/ftadvanc.h
 freetype/ftbitmap.h
 freetype/ftxf86.h
+freetype.h
+ftcache.h
+ftglyph.h
+ftsynth.h
+ftoutln.h
+ttnameid.h
+tttables.h
+t1tables.h
+ftlcdfil.h
+ftsizes.h
+ftadvanc.h
+ftbitmap.h
+ftxf86.h
 fribidi/fribidi.h
 FSp_fopen.h
 fstream
 fstream.h
 ft2build.h
 fts.h
 gconf/gconf-client.h
 Gdiplus.h
diff --git a/js/src/config/system-headers b/js/src/config/system-headers
--- a/js/src/config/system-headers
+++ b/js/src/config/system-headers
@@ -408,16 +408,29 @@ freetype/ftoutln.h
 freetype/ttnameid.h
 freetype/tttables.h
 freetype/t1tables.h
 freetype/ftlcdfil.h
 freetype/ftsizes.h
 freetype/ftadvanc.h
 freetype/ftbitmap.h
 freetype/ftxf86.h
+freetype.h
+ftcache.h
+ftglyph.h
+ftsynth.h
+ftoutln.h
+ttnameid.h
+tttables.h
+t1tables.h
+ftlcdfil.h
+ftsizes.h
+ftadvanc.h
+ftbitmap.h
+ftxf86.h
 fribidi/fribidi.h
 FSp_fopen.h
 fstream
 fstream.h
 ft2build.h
 fts.h
 gconf/gconf-client.h
 Gdiplus.h
